src/index.js

```diff
@@ -1,2 +1,33 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import './style.css';
\ No newline at end of file
+import './style.css';
+import Data from './modules/inputObj.js';
+import submitScore from './modules/submitScore.js';
+import fetchScore from './modules/fetchScore.js';
+
+const formData = document.getElementById('form-box');
+const refreshBTN = document.getElementById('refresh-btn');
+const scoreList = document.getElementById('table');
+
+formData.addEventListener('submit', async (e) => {
+  e.preventDefault();
+
+  const userInput = document.getElementById('playerName');
+  const scoreInput = document.getElementById('points');
```

Comment:
- [ ] Kindly check the `index.html` and add `Id` carefully. the `ID` name inside `getElementById()` should be `scores` not `points`. Fix it kindly using this `const scoreInput = document.getElementById('scores');` I am unable to submit the `Name` and `Score` for this error. Kindly fix this.